core/base/src/TApplication.cxx

```diff
@@ -1059,9 +1060,10 @@ void TApplication::OpenGitHubIssue(const TString &type)
       OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=improvement&template=improvement_report.yml&root-version="+FormatHttpUrl(GetSetup())+"\"");
     } else if (type == "feature") {
     OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=new+feature&template=feature_request.yml\"");
+    Info("OpenGitHubIssue", "A new tab should have opened in your browser.");
```

Comment:
Why only here and not when bug or improvement?

Reply:
because indentation is weird in this function and I can't read code...

Reply:
how about now? :sweat_smile: sorry for all the noise